dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -72,52 +73,39 @@
 ]
 
 
-def cholesky(input):
+def cholesky(a):
```

Comment:
NumPy 2.0 assumes to extend with new keyword `upper` (more details in PR#25388 from [here](https://github.com/numpy/numpy/pulls)) and to align with [Python array API](https://data-apis.org/array-api/draft/extensions/generated/array_api.linalg.cholesky.html#array_api.linalg.cholesky).
I think we need to align here in advance.